.github/workflows/shellspec.yaml

```
@@ -13,34 +13,52 @@ jobs:
         run: |
           echo "${HOME}/.local/bin" >> "$GITHUB_PATH"
           # brew update
           brew install bash
           brew install kcov
+          brew install grep
+          brew install gnu-sed
+          brew install gawk
+          brew install coreutils
           curl -fsSL https://git.io/shellspec | sh -s ${{ env.SHELLSPEC_VERSION }} --yes
       - name: Run shellspec Tests
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2155:warning:1:8: Declare and assign separately to avoid masking return values

(shellcheck)